### ISSUE TITLE
Timeline dates don't match those within the ScheduleView

### ISSUE THREAD

**Issue by nicoulaj** [flex-users/flexlib#87]
Originally filed by slangeb...@gmail.com on 2008-02-19T17:47:07

What steps will reproduce the problem?
1. Look at attached screenshot.
2. If you look at the timeline scrubber at the top, with the dates arranged
above the scrollbar... notice how the dates listed do not match those of
the entries in the schedule itself. EG: The items in the top row start on
1/16, but 1/2/08 doesn't show up in the top timeline labels till way to the
right, almost where the february entries start. 

What is the expected output? What do you see instead?
- Expect dates to match between the two.

What version of the product are you using? On what operating system?
- Winxp - Firefox

Please provide any additional information below.

**Comment by nicoulaj** [comment]
Updated by slangeb...@gmail.com on 2008-02-20T04:44:58

Perceived bug. 

Looks like an alignment issue where date matches vertical line to its left, but is
centered between the two.

Ex1: 10/1/07 matches.
Ex2: Scrolling to right, 4/1/07 still matches.

Recommend:
Looking at style settings for the Timeline control, to move date labels - or find
different layout, such as removing date labels and placing them within the schedule -
etc.

Original ticket set owner to slangeberg
Removed label Type-Defect
Removed label Priority-Medium
Added label Type-Other
Added label Priority-Low
Original ticket set status to WontFix (we converted to closed)
